java/cdk/src/main/java/sleeper/cdk/stack/CompactionStack.java

```diff
@@ -261,7 +258,7 @@ private void lambdaToFindCompactionJobsThatShouldBeCreated(CoreStacks coreStacks
 
         IFunction handler = jobCreatorJar.buildFunction(this, "JobCreationLambda", builder -> builder
                 .functionName(functionName)
-                .description("Scan DynamoDB looking for files that need merging or splitting and create appropriate job specs in DynamoDB")
+                .description("Scan DynamoDB looking for files that need merging and create appropriate job specs in DynamoDB")
```

Comment:
Change 'merging' to 'compacting' for consistency.

Reply:
Done, thanks